### PR TITLE
Modified alerting logic to handle sds native alerts also

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
tendrl-bug-id: Tendrl/node-agent#617

Signed-off-by: GowthamShanmugam <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
@shtripat @nnDarshan @r0h4n please review this

File: tendrl/node_agent/alert/__init__.py [hunk 1]
```diff
@@ -41,55 +41,60 @@ def update_alert(message):
...
-                "error",
-                NS.publisher_id,
-                {
-                    "message": "New alert can't be a clearing alert %s" % (
-                        new_alert
-                    )
-                }
-            )
+            # SDS native events
+            utils.save_utilization_only_alert(new_alert_obj)
```
Comment (r0h4n) [inline tendrl/node_agent/alert/__init__.py]: save utilization of what? This method name is confusing, simplify it and add helping comment

Comment (shtripat) [reply tendrl/node_agent/alert/__init__.py]: `save_notification_only_alert`. My bad for creating confusion.

**Comment by GowthamShanmugam** [comment]
@shtripat @r0h4n changes done

File: tendrl/node_agent/alert/__init__.py [hunk 2]
```diff
@@ -41,55 +41,60 @@ def update_alert(message):
...
-                        utils.classify_alert(new_alert_obj)
-                        new_alert_obj.save()
+                        if message.payload["alert_condition_unset"]:
+                            keep_alive = int(
+                                NS.config.data["alert_retention_time"]
+                            )
+                            utils.classify_alert(new_alert_obj, keep_alive)
+                            new_alert_obj.save(ttl=keep_alive)
+                        else:
+                            # Remove the clearing alert with same if exist
```
Comment (shtripat) [inline tendrl/node_agent/alert/__init__.py]: Can you explain why this delete and then save is required in comments?

**Pull request merged**